src/main/java/io/stargate/sgv2/jsonapi/api/v1/CollectionResource.java

```diff
@@ -220,17 +220,16 @@ public Uni<RestResponse<CommandResult>> postCommand(
                 }
                 // otherwise use generic for now
                 return Uni.createFrom().item(new ThrowableCommandResultSupplier(error));
-
               } else {
-
                 // TODO No need for the else clause here, simplify
-                var apiFeatures =
-                    ApiFeatures.fromConfigAndRequest(
-                        apiFeatureConfig, requestContext.getHttpHeaders());
-                if ((schemaObject.type() == SchemaObject.SchemaObjectType.TABLE)
-                    && !apiFeatures.isFeatureEnabled(ApiFeature.TABLES)) {
-                  return Uni.createFrom()
-                      .failure(ErrorCodeV1.TABLE_FEATURE_NOT_ENABLED.toApiException());
+                if (schemaObject.type() == SchemaObject.SchemaObjectType.TABLE) {
```

Comment:
Minor refactoring to only access `ApiFeatures` when we actually need them (for Table command).